libraries/botbuilder-azure-blobs/tests/blobsStorage.test.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
```

Comment:
These shouldn't be required with the updated config in root, have you recently tried removing these `eslint-disable` lines?